src/external_libs/Makefile.am

```diff
@@ -11,12 +11,18 @@ CLEANFILES = .libcdada_mark
 ## LIBCDADA
 ##
 _libcdada:
-	@if [ -z "`git rev-parse HEAD 2> /dev/null`" ]; then \
-		echo "[dep: libcdada] Skipping, not a git repository!"; \
-		exit 0;\
+	@not_git=0; \
+	if [[ -z "$(shell git rev-parse HEAD 2> /dev/null)" ]]; then \
```

Comment:
Definetely not portable to BSD, both `$(shell)` and the use of `[[ ]]`. Revert to original:

```
if [ -z "`git rev-parse HEAD 2> /dev/null`" ]; then \
```